src/components/Filters/FilterFile/UploadFile/FileUploaderField.js

```diff
@@ -31,17 +25,17 @@ const FileUploaderField = ({
     }
   }, [value, file]);
 
-  const processError = (resp, intl) => {
+  const processError = (resp) => {
     const contentType = resp.headers ? resp.headers.get('Content-Type') : '';
 
     if (contentType.startsWith('application/json')) {
       throw new Error(`${resp.message} (${resp.error})`);
     } else {
-      throw new Error(intl.formatMessage({ id: 'errors.uploadError' }));
+      throw new Error('uploadError');
```

Comment:
untranslated string value

Reply:
Since there was never a translation available for `'errors.uploadError'` its not changing anything.
Do you want me to add a translation for the token 'errors.uploadError'?